bin/init.sh

```diff
@@ -9,7 +19,7 @@ if [ $PDIR != "$GOPATH/src/istio.io/pilot" ]; then
 fi
 
 # Building and testing with Bazel
-bazel build //...
+bazel build ${bazel_build_args[@]} //...
```

Comment:
If you really need to customize bazel build, then I suggest remove this line from this script.